src/main/java/it/gov/pagopa/pu/debtpositions/util/InstallmentUtils.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.pu.debtpositions.util;
+
+import it.gov.pagopa.pu.debtpositions.dto.generated.InstallmentStatus;
+import it.gov.pagopa.pu.debtpositions.model.InstallmentNoPII;
+import it.gov.pagopa.pu.debtpositions.model.InstallmentSyncStatus;
+import java.util.Set;
+
+public class InstallmentUtils {
+
+  private InstallmentUtils() {
+  }
+
+  public static final Set<String> TRANSITION_TO_SYNC_REQUIRED = Set.of(
+    InstallmentStatus.DRAFT + "|" + InstallmentStatus.UNPAID,
+    InstallmentStatus.UNPAID + "|" + InstallmentStatus.INVALID,
+    InstallmentStatus.UNPAID + "|" + InstallmentStatus.CANCELLED);
+
+  public static final Set<InstallmentStatus> NOT_PAID = Set.of(
+    InstallmentStatus.DRAFT,
+    InstallmentStatus.TO_SYNC,
+    InstallmentStatus.UNPAID,
+    InstallmentStatus.EXPIRED);
+
+
+  public static boolean isTransitionToSync(InstallmentStatus statusFrom,
+    InstallmentStatus statusTo) {
+    return TRANSITION_TO_SYNC_REQUIRED.contains(statusFrom + "|" + statusTo);
+  }
+
+  public static void updateInstallmentFields(InstallmentNoPII installment, InstallmentStatus status) {
```

Comment:
this is not a generic update
```suggestion
  public static void setStatus(InstallmentNoPII installment, InstallmentStatus status) {
```

Reply:
let's add a javadoc:

> It will set the Installment status verifying it will need the TO_SYNC transition